javascript/package.json

```diff
@@ -26,7 +26,9 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom-fourteen",
     "coverage": "react-scripts test --env=jsdom-fourteen --coverage --watchAll=false",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "storybook": "start-storybook -p 6006 -s public",
+    "build-storybook": "build-storybook -s public"
```

Comment:
We might want to add the output `public` directory into the `.gitignore` file so we don't accidentally check in a static storybook into our repo.

Reply:
Done